<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>

  <include file="$(find my_patrol_sim)/launch/includes/stage_and_map.launch.xml">
    <arg name="map_name" value="cumberland" />
  </include>

  <include file="$(find my_patrol_sim)/launch/includes/one_robot.launch.xml">
    <arg name="robot_name" value="robot_0" />
  </include>
  <include file="$(find my_patrol_sim)/launch/includes/one_robot.launch.xml">
    <arg name="robot_name" value="robot_1" />
  </include>
  <include file="$(find my_patrol_sim)/launch/includes/one_robot.launch.xml">
    <arg name="robot_name" value="robot_2" />
  </include>
  <include file="$(find my_patrol_sim)/launch/includes/one_robot.launch.xml">
    <arg name="robot_name" value="robot_3" />
  </include>

</launch>